src/Forms/GridField/GridFieldVersionedState.php

```diff
@@ -11,7 +11,7 @@
 
 
 /**
- * @deprecated 4.1.0:5.0.0
+ * @deprecated 4.1.0
```

Comment:
Add the message here and update the version in the notice below.

Reply:
Updated